sdk/easm/ci.yml

```diff
@@ -0,0 +1,35 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - main
+    - hotfix/*
+    - release/*
+    - restapi*
+  paths:
+    include:
+    - sdk/easm/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+    - restapi*
```

Comment:
We treat feature/hotfix/release branch prefixes as a special case. Do we need `restapi` as a branch prefix or will feature be sufficient?

Reply:
I can change it, but it's in there because that's what's in [`./scripts/quickstart_tooling_dpg/template_ci/ci.yml`](https://github.com/Azure/azure-sdk-for-python/blob/main/scripts/quickstart_tooling_dpg/template_ci/ci.yml)